master/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -20,16 +20,13 @@ system.
 
 ## Before you begin 
 
-- Ensure that your cluster meets the [requirements](../../../reqs/system).
-  An easy way to create a cluster which meets these requirements is by 
+- Ensure that your cluster meets the {{site.prodname}} [System requirements](../../../reqs/system).
```

Comment:
lowercase system?

Reply:
It is meant to quote the title of the page, which has an uppercase System 😄